xapian-bindings/python3/README

```diff
@@ -0,0 +1,29 @@
+Building a Python package
+=========================
+
+First, build xapian-core for static linking:
+
+$ cd xapian-core
+$ ./configure --enable-static --disable-shared --enable-pic [--enable-maintainer-mode] (when building from git)
+$ make
+
+Install the required Python packages:
+
+$ pip install sphinx build
+
+Then build the Python module:
+
+$ cd xapian-bindings
+$ XAPIAN_CONFIG=../xapian-core/xapian-config ./configure --with-python3 [--enable-maintainer-mode] (when building from git)
+$ cd python3
+$ make
```

Comment:
Again, if building from git people should use the top-level `configure` and are probably better advised to build from a release tarball.

If you're building from a release tarball it's not going to be `../xapian-core` unless you carefully rename things.